seatunnel-translation/seatunnel-translation-base/src/main/java/org/apache/seatunnel/translation/source/CoordinatedSource.java

```diff
@@ -143,7 +153,9 @@ public void open() throws Exception {
                 .forEach(
                         entry -> {
                             try {
+                                // Initialize reader
                                 entry.getValue().open();
+                                // Allocates split whose status is pending
```

Comment:
Would that be a better way to describe it ？"the corresponding status is assigned to the reader"